src/libstd/process.rs

```diff
@@ -1080,6 +1080,15 @@ impl fmt::Display for ExitStatus {
     }
 }
 
+/// This is ridiculously unstable, as it's a completely-punted-upon part
```

Comment:
I don't really care, but I feel like this is overstating the case. Specifying an exit code doesn't seem so "wild and crazy" to me.

Reply:
That's not what I meant, but I can see what you mean after rereading.  I'll make a follow-up PR weakening the language here.

I definitely agree that `fn main() -> ExitCode` is valuable.  I only intended the statement to be about the precise form of this type, since it's not <kbd>B-RFC-Approved</kbd> and thus even more subject to change than usual.